src/index.md

```diff
@@ -11,15 +11,20 @@ import { combine, move, clone } from "./utils/ratios-utils.js";
 const SparkRates = FileAttachment("./data/spark-rsr.json").json();
 const SparkNonZeroRates = FileAttachment("./data/spark-rsr-non-zero.json").json();
 const SparkMinerRates = FileAttachment("./data/spark-miners-rsr.json").json();
+const SparkMinerRetrievalTimings = FileAttachment("./data/spark-miners-retrieval-timings.json").json();
 const SparkRetrievalResultCodes = FileAttachment("./data/spark-retrieval-result-codes.json").json();
 const SparkMinerRsrSummaries = FileAttachment("./data/spark-miner-rsr-summaries.json").json();
+const SparkRetrievalTimes = FileAttachment("./data/spark-retrieval-timings.json").json();
 ```
 
 ```js
 const nonZeroSparkMinerRates = SparkMinerRates.filter((record) => record.success_rate != 0)
 const tidySparkMinerRates = SparkMinerRates
   .sort((recordA, recordB) => recordB.success_rate - recordA.success_rate)
-  .map(record => ({ ...record, success_rate: `${(record.success_rate * 100).toFixed(2)}%`,success_rate_http: `${(record.success_rate_http * 100).toFixed(2)}%`}))
+  .map(record => {
+    const { ttfb_ms } = SparkMinerRetrievalTimings.find((ttfbRecord) => ttfbRecord.miner_id === record.miner_id) ?? {};
```

Comment:
Are you concerned with the performance of this line? If yes, would it be worth to add an index `ttfbRecordByMinerId`?

Reply:
Good suggestion. I wasn't particularly concerned about performance at the time I was building this because search is happening during build time but the fix was quick and easy so it should be good now.